apps/modernization-ui/src/apps/patient/profile/PatientProfile.tsx

```diff
@@ -137,28 +137,40 @@ export const PatientProfile = () => {
                 <div className="main-body">
                     {patient && summary && <PatientProfileSummary summary={summary} patient={patient} />}
 
-                    <div className="grid-row flex-align-center">
-                        <h6
+                    <div role="tablist" className="grid-row flex-align-center">
+                        <p
+                            role="tab"
+                            aria-labelledby="summary-tabpanel"
```

Comment:
question:  isn't this the label for the tab?  not sure if an `aria-labeledby` is needed here

Reply:
> question: isn't this the label for the tab? not sure if an `aria-labeledby` is needed here

Button was added so it's possible to change tabs with enter button. Was able to get exact desired design besides blue hover.
<img width="1229" alt="Screenshot 2024-02-02 at 9 30 04 AM" src="https://github.com/CDCgov/NEDSS-Modernization/assets/125376056/592af34e-dee2-4d44-b1bc-d261a6c10388">

Reply:
> question: isn't this the label for the tab? not sure if an `aria-labeledby` is needed here

I followed tabpanel https://www.w3.org/WAI/ARIA/apg/patterns/tabs/ 

<img width="1020" alt="Screenshot 2024-02-02 at 9 51 11 AM" src="https://github.com/CDCgov/NEDSS-Modernization/assets/125376056/0121e527-3357-4b96-9a63-10440749a864">

"Each element with role tabpanel has the property [aria-labelledby](https://w3c.github.io/aria/#aria-labelledby) referring to its associated tab element."